### PR TITLE
kernelci.org: add Advisory Board role definitions

### PR DESCRIPTION
**Pull request by gctucker**
Add a section on the Advisory Board page with definitions for the key roles (Chair, Treasurer, PM).  Also update the wording in the intro paragraph accordingly.

### CODE REVIEW

File: kernelci.org/content/en/docs/org/board.md [hunk 1]
```diff
@@ -11,11 +11,13 @@ The Advisory Board (AB) is composed of one representative from each premium
...
-year and voted by the board.
+By the end of December each year, the Board agrees on a budget for the next
+year and submits it to the Linux Foundation.  It then elects a Chair, a
+Treasurer and a General Members representative for the following year.  The new
+roles and budget take effect on January 1st.
 
-There is also a Project Manager role to help coordinating things with the wider
-team.
+In addition to the elected roles, there is also an appointed Project Manager to
+help coordinate activites with the TSC, working groups and the wider team.
```
Comment (patersonc) [inline kernelci.org/content/en/docs/org/board.md:20]:
```suggestion
help coordinate activities with the TSC, working groups and the wider team.
```

File: kernelci.org/content/en/docs/org/board.md [hunk 2]
```diff
@@ -31,6 +33,74 @@ their respective roles, email address and IRC nicknames:
...
+### Representatives
+
+Other voting members of the Board include:
+
+* Premium Member representatives
+* General Member representatives
+* TSC representative
+
+They are all expected to cast a vote and take part in the Board meetings to
+take initiatives and help the project strive.
```
Comment (patersonc) [inline kernelci.org/content/en/docs/org/board.md:106]:
```suggestion
take initiatives and help the project thrive.
```

**Review by patersonc — CHANGES_REQUESTED** [review]
Thank you @gctucker

File: kernelci.org/content/en/docs/org/board.md [hunk 3]
```diff
@@ -31,6 +33,74 @@ their respective roles, email address and IRC nicknames:
...
+### Chair
+
+As per the Funding Charter provided by the Linux Foundation:
+
+> The Chair will preside over meetings of the Governing Board, manage any
+> day-to-day operational decisions, and will submit minutes for Governing Board
+> approval.
+
+Then in the particular case of KernelCI, past discussions have led the Board to
+agreed on the following responsibilities:
```
Comment (patersonc) [inline kernelci.org/content/en/docs/org/board.md:55]:
```suggestion
agree on the following responsibilities:
```

**Comment by patersonc** [comment]
Other than the small correction above, this MR looks okay to me.

**Comment by gctucker** [comment]
Thanks for the review, I've applied your typo fix.

**Review by patersonc — APPROVED** [review]

**Pull request merged**